src/nvim/window.c

```diff
@@ -1044,19 +1044,22 @@ static void win_init_some(win_T *newp, win_T *oldp)
 }
 
 
-/*
- * Check if "win" is a pointer to an existing window.
- */
-int win_valid(win_T *win)
+/// Return true if "win" is a pointer to an existing window in the current
+/// tabpage.
```

Comment:
“return” should be in `@return`. Original summary was better suited to being a summary.